calico/felix/config.py

```diff
@@ -346,11 +348,9 @@ def _finish_update(self, final=False):
         # Now the config has been validated, generate the IPTables mark masks
         # we'll actually use internally.
         mark_mask = self.IPTABLES_MARK_MASK
-
-        # Extract the least significant bit and use it as the accept mask.
```

Comment:
I think worth retaining and adding to that comment: the ACCEPT bit is the least significant, and the NEXT_TIER bit is the next least significant.